app/src/main/java/com/example/myapplication/utils/Utils.kt

```diff
@@ -1,8 +1,12 @@
 package com.example.myapplication.utils
 
-const val EMPTY_STRING : String = ""
-const val NULL_RESULT : Float = 0F
-const val PLUS : String = "+"
-const val SUBTRACT : String = "-"
-const val DIVIDE : String = "/"
-const val MULTIPLY : String = "x"
+const val EMPTY_STRING: String = ""
+const val ZERO_FLOAT_RESULT: Float = 0F
+const val PLUS: String = "+"
+const val SUBTRACT: String = "-"
+const val DIVIDE: String = "/"
+const val MULTIPLY: String = "x"
+const val TOAST_ZERO_DIVIDE: String = "MATH ERROR"
```

Comment:
TOAST_ZERO_DIVIDE is not meaningful enough, it could be something related to `ERROR_MESSAGE_DIVIDE_BY_ZERO`